autoortho/downloader.py

```diff
@@ -229,8 +229,8 @@ def _show_progress(self, block_num, block_size, total_size):
         MBps = (total_fetched/1048576) / elapsed
         cur_activity['pcnt_done'] = pcnt_done
         cur_activity['MBps'] = MBps
-        print(f"\r{pcnt_done:.2f}%   {MBps:.2f} MBps", end='')
-        cur_activity['status'] = f"Downloading {self.dl_url}\n{pcnt_done:.2f}%   {MBps:.2f} MBps"
+        print(f"\r{pcnt_done:.2f}%   {MBps:.2f} Mbps", end='')
+        cur_activity['status'] = f"Downloading {self.dl_url}\n{pcnt_done:.2f}%   {MBps:.2f} Mbps"
```

Comment:
Agree this is confusing.  I think the rest of the references should also be updated to reflect this.

Also, I think the lines:

```
        cur_activity['pcnt_done'] = pcnt_done
        cur_activity['MBps'] = MBps
```

Can go away if the corresponding lines (350,351) in config_ui.py are removed as well.